deployment/installInsightAgent.py

```diff
@@ -108,8 +113,12 @@ def sshInstallHypervisor(retry,hostname):
     except socket.error, e:
         print "Socket connection failed in %s:"%hostname, e
         return sshInstallHypervisor(retry-1,hostname)
-    #except:
-        #print "Unexpected error in %s:"%hostname
+    except IOError,e :
+        print "Not enough disk space in host"
```

Comment:
Thats correct. The most common problem when deploying in hypervisor hosts is that there are not enough disk space. We want to specifically capture that. Eventhough, we are printing "Not enough disk space in host" here, this will be captured by deployInsightAgent.py since it is calling installInsightAgent.py in subprocess. In deployInsightAgent.py, the error message is printed.